src/ui/server/controllers/insurance/business/turnover/validation/rules/estimated-annual-turnover.ts

```diff
@@ -28,7 +28,8 @@ const estimatedAnnualTurnover = (responseBody: RequestBody, errors: object) => {
   }
 
   const errorMessage = ERROR_MESSAGE.INCORRECT_FORMAT;
-  return wholeNumberValidation(responseBody, errors, errorMessage, FIELD_ID);
+  const allowNegativeValue = true;
```

Comment:
Very minor and not a blocker but can we add a space between these 3 const's so it's easier to read